combine_ucr.py

```diff
@@ -135,13 +136,13 @@ def parse_arguments():
                 )
                 grid_search.fit(split[0], split[2])
             classifier = grid_search.best_estimator_
-        sklearn.externals.joblib.dump(
+        joblib.dump(
```

Comment:
I would prefer keeping the sklearn.externals version of joblib. With your code it seems impossible to load my pretrained models.

Reply:
I have reverted changes regarding joblib. However, it is very strange that you cannot load you the pretrained models. I have checked and haven't discovered any changes in the default serialization protocols.  Could you please provide me an example of a pretrained model.

Reply:
Hi, sorry for the very late reply! I added an example of a pretrained model in the repository in case you want to check.